### ISSUE TITLE
Error in Hosts->Settings-Log

### ISSUE THREAD

**Issue by PeterVoronov** [ioBroker/ioBroker.admin#1327]
**Describe the bug**  
I can't enter in LOG configuration for host. Previously it was SEQ target is configured.

**To Reproduce**  
In my case - to setup SEQ channel for the logs on previous version of admin, and try to enter in this page now.

**Expected behavior**  
It have to be accessible and work properly

**Screenshots & Logfiles**  
![iobroker admin log-error](https://user-images.githubusercontent.com/66964358/155295733-bfaaa078-4693-4f93-abe3-f6af900e650e.png)
and error in Console
```
instrument.ts:125 
        
       TypeError: Cannot read properties of null (reading 'includes')
    at BaseSettingsLog.js:77:65
    at Array.forEach (<anonymous>)
    at new s (BaseSettingsLog.js:75:41)
    at zc (react-dom.production.min.js:135:172)
    at Uo (react-dom.production.min.js:181:146)
    at Zi (react-dom.production.min.js:269:427)
    at jl (react-dom.production.min.js:250:347)
    at Sl (react-dom.production.min.js:250:278)
    at Cl (react-dom.production.min.js:250:138)
    at zl (react-dom.production.min.js:243:163)
    at react-dom.production.min.js:123:115
    at t.unstable_runWithPriority (scheduler.production.min.js:18:343)
    at Ga (react-dom.production.min.js:122:325)
    at Ka (react-dom.production.min.js:123:61)
    at Za (react-dom.production.min.js:122:428)
    at ke (react-dom.production.min.js:292:101)
    at Yt (react-dom.production.min.js:73:352)
    at HTMLBodyElement.a (helpers.ts:98:17)
```

**Versions:**  
 - Adapter version: v5.3.1
 - JS-Controller version: 4.0.15
 - Node version: v14.18.2
 - Operating system: Debian 11

**Additional context**  
I'm not sure, from which version of admin this bug ...

**Comment by Feuer-sturm** [comment]
I have the same issue with admin 5.3.1
My log in browser console
![grafik](https://user-images.githubusercontent.com/25422320/155800215-bfb9b0dd-67f1-4ed3-932b-e8b7c7ad7a95.png)

**Comment by Lyxon1337** [comment]
same issue

docker container v6.1.0
Admin 5.3.1
JS-Controller 4.0.18
node.js 14.19.0
npm 6.14.16

![grafik](https://user-images.githubusercontent.com/91346264/157156956-db288f12-a0eb-4531-8d48-62ae9c4b16c4.png)

**Comment by Lyxon1337** [comment]
same issue

docker container v6.1.0
build 2022-03-07T23:45:56+00:00
Admin 5.3.1
JS-Controller 4.0.19
node.js 14.19.0
npm 6.14.16

![grafik](https://user-images.githubusercontent.com/91346264/157227730-47680a4f-5bcc-41d7-809f-106f09bedc96.png)

**Comment by Feuer-sturm** [comment]
Is fixed with Admin 5.3.2

**Comment by Feuer-sturm** [comment]
@Apollon77 you can unpin this issue

**Comment by Apollon77** [comment]
I will leave until in stable

**Comment by PeterVoronov** [comment]
> Is fixed with Admin 5.3.2

thanks. can confirm

**Comment by Feuer-sturm** [comment]
@Apollon77 I think you can unpin this issue.
